dll/win32/shell32/lang/pt-PT.rc

```diff
@@ -46,15 +46,15 @@ MENU_003 MENU
 BEGIN
     MENUITEM "&Retroceder\tCtrl+Z", FCIDM_SHVIEW_UNDO
     MENUITEM SEPARATOR
-    MENUITEM "Cor&tar\tCtrl+X", FCIDM_SHVIEW_CUT //FIXME: accelerator collision &T
+    MENUITEM "Cor&tar\tCtrl+X", FCIDM_SHVIEW_CUT
     MENUITEM "&Copiar\tCtrl+C", FCIDM_SHVIEW_COPY
-    MENUITEM "Co&lar\tCtrl+V", FCIDM_SHVIEW_INSERT
-    MENUITEM "Colar &atalho", FCIDM_SHVIEW_INSERTLINK
+    MENUITEM "C&olar\tCtrl+V", FCIDM_SHVIEW_INSERT
+    MENUITEM "Colar atal&ho", FCIDM_SHVIEW_INSERTLINK
```

Comment:
Your PR now looks much better already. Still one last question. You do use here for edit menu at line 51:
```
    MENUITEM "C&olar\tCtrl+V", FCIDM_SHVIEW_INSERT
    MENUITEM "Colar atal&ho", FCIDM_SHVIEW_INSERTLINK
```

Is it ok and intended that different accelerators are used for the "shellview background menu" at line 39?
```
        MENUITEM "Co&lar", FCIDM_SHVIEW_INSERT
        MENUITEM "Colar a&talho", FCIDM_SHVIEW_INSERTLINK
```

Please either harmonize that, or show a screenshot of 2k3sp2pt-pt that this is ok. I will approve and merge your PR afterwards.